### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by dalpo**
Update changelog for the related PR https://github.com/activerecord-hackery/ransack/pull/604

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 ## Unreleased
 ### Added
 
+*   Added optional block argument for the `sort_link` method. PR
+    [#604](https://github.com/activerecord-hackery/ransack/pull/604), commit
+    [2e6667b](https://github.com/dalpo/ransack/commit/2e6667b).
```
Comment (jonatack) [inline CHANGELOG.md]: Should be commit 997b856 here.

**Comment by jonatack** [comment]
Thanks! Could you update the commit and add a quick update to the sort_link section of the README?

**Comment by jonatack** [comment]
Thanks! Could you update the commit link and add a quick update to the sort_link section of the README?

Also, please add [skip ci] to your commit message so the whole travis-ci doesn't :runner: needlessly for doc-only changes.

**Comment by dalpo** [comment]
@jonatack Sorry for my late response.

I made some update to the changelog and the readme here https://github.com/activerecord-hackery/ransack/pull/633
Feel free to update my bad english :smile: 

Thanks!

**Pull request merged**